sbi/inference/base.py

```diff
@@ -491,6 +493,8 @@ def _summarize(
             global_step=round_ + 1,
         )
 
+
```

Comment:
Please run `black`

Reply:
Would love to do so, but just calling `black` resulted in tons of changes to the code base. Do you mean to run `black` on the entire repo or just the files I touch in this PR?

Reply:
I followed the contribution guidelines (on `black` and `isort`) and now see more files changed than I touched:
```
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   sbi/analysis/conditional_density.py
	modified:   sbi/inference/base.py
	modified:   sbi/inference/posteriors/base_posterior.py
	modified:   sbi/inference/posteriors/likelihood_based_posterior.py
	modified:   sbi/inference/posteriors/ratio_based_posterior.py
	modified:   sbi/inference/snle/snle_base.py
	modified:   sbi/inference/snpe/snpe_base.py
	modified:   sbi/inference/snpe/snpe_c.py
	modified:   sbi/inference/snre/snre_base.py
	modified:   sbi/utils/user_input_checks.py
	modified:   sbi/utils/user_input_checks_utils.py
	modified:   tests/user_input_checks_test.py
```
:shrug: Should I add/commit all of this?

Reply:
That's very strange. Thanks for checking! Please do not commit this, we'll have to check our black settings (after this PR is merged)

Reply:
:face_with_head_bandage: ok, I know what happened: I ran `black .` in a subdir of the repo root. Then it apparently doesn't pick up the configurations from [setup.cfg](/mackelab/sbi/setup.cfg)

Reply:
then I tried again running `black` from the repo root (after a `git restore`), but to my surprise I get:
```
$ black .               
reformatted /path/development/sbi/sbi/inference/__init__.py
reformatted /path/development/sbi/sbi/analysis/conditional_density.py
reformatted /path/development/sbi/sbi/inference/posteriors/likelihood_based_posterior.py
reformatted /path/development/sbi/sbi/inference/posteriors/ratio_based_posterior.py
reformatted /path/development/sbi/sbi/inference/snpe/snpe_base.py
reformatted /path/development/sbi/sbi/inference/base.py
reformatted /path/development/sbi/sbi/inference/snpe/snpe_c.py
reformatted /path/development/sbi/sbi/utils/user_input_checks_utils.py
reformatted /path/development/sbi/sbi/inference/posteriors/base_posterior.py
reformatted /path/development/sbi/tests/mcmc_slice_pyro/test_slice.py
All done! ✨ 🍰 ✨
10 files reformatted, 77 files left unchanged.
```
Most of these files I didn't touch during this PR. Should I proceed?

```
$ black --version
black, version 20.8b1
```

Reply:
Both `isort` and `black` uncover fixes in files that I haven't touched with this PR. How to proceed?